src/i_video.c

```diff
@@ -497,14 +497,22 @@ static void MapMouseWheelToButtons(SDL_MouseWheelEvent *wheel)
     static event_t down;
     int button;
 
-    if (wheel->y <= 0)
+    if (wheel->y < 0)
```

Comment:
We'll lose the hypothetical (maybe not so?) case that `wheel` is all zero here. Please do either initialize `button` to something reasonable, or keep the `<=` checks (preferred) or add an `else return` clause.

Reply:
No, wait. Please add an `else return` clause.